From randy.dunlap@oracle.com  Mon Jul 27 09:36:40 2009
From: Lucian Adrian Grijincu <lgrijincu@ixiacom.com>
Date: Mon, 27 Jul 2009 09:06:42 -0700
Subject: driver core: documentation: make it clear that sysfs is optional
Cc: gregkh@suse.de, Lucian Adrian Grijincu <lgrijincu@ixiacom.com>
Message-ID: <20090727090642.0e495919.randy.dunlap@oracle.com>


From: Lucian Adrian Grijincu <lgrijincu@ixiacom.com>

The original text suggested that sysfs is mandatory and always
compiled in the kernel.

Signed-off-by: Lucian Adrian Grijincu <lgrijincu@ixiacom.com>
Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/filesystems/sysfs.txt |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/Documentation/filesystems/sysfs.txt
+++ b/Documentation/filesystems/sysfs.txt
@@ -23,7 +23,8 @@ interface. 
 Using sysfs
 ~~~~~~~~~~~
 
-sysfs is always compiled in. You can access it by doing:
+sysfs is always compiled in if CONFIG_SYSFS is defined. You can access
+it by doing:
 
     mount -t sysfs sysfs /sys 
 
